pom.xml

```diff
@@ -177,12 +166,6 @@
       <artifactId>rest-assured</artifactId>
       <version>4.3.0</version>
       <scope>test</scope>
-      <exclusions>
-        <exclusion>
-          <groupId>commons-logging</groupId>
-          <artifactId>commons-logging</artifactId>
-        </exclusion>
-      </exclusions>
```

Comment:
This logging library now seems to be required by restassured-it will not compile without it.